swiss-ng-api.yaml

```diff
@@ -6255,8 +6262,9 @@ components:
 
       type: object
       required:
+        - endToEndIdentification
         - debtorAccount
-        - instructedAmount
+        - debtorName
```

Comment:
According to the table in the description, the `debtorName` is not mandatory in any case.